packages/web-components/src/menu-item/menu-item.ts

```diff
@@ -95,11 +140,15 @@ export class MenuItem extends FASTElement {
    */
   protected slottedSubmenuChanged(prev: HTMLElement[] | undefined, next: HTMLElement[]) {
     this.submenu?.removeEventListener('toggle', this.toggleHandler);
+    this.elementInternals.ariaHasPopup = null;
+    toggleState(this.elementInternals, 'submenu', false);
```

Comment:
We appear to be setting some things multiple times, should we instead put some of these items into an `else` block below?

Reply:
I can move both of these